pkg/cmd/root.go

```diff
@@ -8,12 +8,11 @@ import (
 )
 
 var basePath string
+var outputFormat string
```

Comment:
It initially confused me as to where this variable had come from. I feel like some sort of prefix should be applied to global options in order to clarify their purpose.